### ISSUE TITLE
Extensions: Details, Contibution, Changelog, Dependencies are not reachable by tab

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/vscode#61706]
**Environment Details:**
VSCode Version : 1.25.1
OS Version : Win10
MAS2.4.3
**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "Extension" button of "ActivityBar"->Enter "PowerShell" in "Search Extension" text field and click on "Install" button.
3.Navigate through keyboard such as "Tab" key or arrow keys to the Tab items "Details,Contributions,Changelog,Dependencies".
4.Observed that the keyboard navigation that whether the keyboard focus goes to the tab item which is activated.

**Actual:**
Keyboard focus does not go to item which is enabled.

**Expected:**
Keyboard focus should go to the tab item which is enabled through Tab or arrow keys.
Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will get confuse that the tab focus skips the tab item.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference**
[2726897_A11y_PowershellEditorServices_Installation_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2510317/2726897_A11y_PowershellEditorServices_Installation_Keyboard.pptx)

**Comment by isidorn** [comment]
Should be fxied in latest, try with latest vscode insiders

**Comment by isidorn** [comment]
The issue here is that the details in the Extension Detail page do not have correct tab order.
Reopening and assigning to @sandy081 to investigate

![image](https://user-images.githubusercontent.com/1926584/56122164-09647600-5f72-11e9-9add-f2c058233a88.png)

**Comment by isidorn** [comment]
Let's close as dupe of #61616
And have that all together
